src/test/ui/conflicting-repr-hints.stderr

```diff
@@ -1,18 +1,18 @@
 error[E0566]: conflicting representation hints
-  --> $DIR/conflicting-repr-hints.rs:13:8
+  --> $DIR/conflicting-repr-hints.rs:19:8
```

Comment:
I am struggling to work out why the order of these errors is swapped... It doesn't seem like the end of the world, but it's quite strange and a little disconcerting not knowing...